src/test/java/wanya/ToDoTest.java

```diff
@@ -0,0 +1,26 @@
+package wanya;
+
+import wanya.task.ToDo;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ToDoTest {
+    @Test
+    public void todoToStringTest() {
+        ToDo todo = new ToDo ("Homework", true);
+        assertEquals("[T][X] Homework", todo .toString());
+
+        ToDo todo2 = new ToDo ("Homework2", false);
+        assertEquals("[T][ ] Homework2", todo2.toString());
+    }
+
+    @Test
+    public void todoToStorageTest() {
```

Comment:
I would like to commend you on how you have strictly followed the Java coding standard of using Egyptian, K&R style brackets throughout your entire code.